iosApp/PhongKMMIC/Sources/Supports/Helpers/UITests/UITestHelper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UITestHelper.swift
+//  Survey
+//
+//  Created by Bliss on 7/12/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import UIKit
+
+final class UITestHelper {
+
+    static let shared = UITestHelper()
+
+    var speed: Float = 1.0
+
+    var isFastAnimationUITests: Bool {
+        LaunchArgument.contains(.fastAnimation)
+    }
+
+    func speedUpUITestAnimation() {
+        #if DEBUG
```

Comment:
By doing this
```
 #if DEBUG
    UITestHelper.shared.speedUpUITestAnimation()
#endif
```
I think we don't need to use `#if DEBUG` in here anymore, what do you think?

Reply:
I think it is better to keep that in `AppDelegate` it easier to read and immediately acknowledge that function only run on DEBUG